lib/main.coffee

```diff
@@ -188,6 +188,32 @@ module.exports =
         return p
     return execPath
 
+  getProjectPath: (lintedFile) ->
+    projectPaths = atom.project.getPaths()
+    projectNames = projectPaths
+      .map (projectPath) -> (
+        splittedPath = projectPath.split('/')
+        return splittedPath[splittedPath.length - 1]
+      )
+
+    projectName = projectNames
+      .find((projectName) -> lintedFile.indexOf(projectName) > -1)
+
+    return projectPaths[projectNames.indexOf(projectName)]
+
+
+  getExistingConfigFiles: (configFiles, projectPath) ->
+    return configFiles
+      .split(/[ ,]+/)
+      .map (configFile) -> path.join(projectPath, configFile)
```

Comment:
Check out the `helpers.findCached()` function, you pass it a directory and a (an array of) name(s) and it will find them for you.